python/katana/local/import_data.py

```diff
@@ -66,6 +75,29 @@ def from_edge_list_arrays(
     """
     Convert an edge list represented as two parallel arrays into a :py:class:`~katana.local.Graph`.
 
+    This preserves node IDs, but **not** edge IDs.
+    """
+    return _from_edge_list_arrays_impl(sources, destinations, property_dict, edges_sorted=False, properties=properties)
+
+
+def from_sorted_edge_list_arrays(
+    sources: np.ndarray, destinations: np.ndarray, property_dict: Dict[str, np.ndarray] = None, **properties: np.ndarray
+) -> Graph:
+    """
+    Convert an **sorted** edge list represented as two parallel arrays into a :py:class:`~katana.local.Graph`. The
+    ``sources`` array must be sorted or the resulting graph will be incorrect.
```

Comment:
Why do we need a separate path for sorted?

Reply:
I debated on this too. I decided two functions was best because they have different semantics. One call maintains edge IDs the other does not. So it's not just a performance different, it's a concrete correctness difference. And forcing the users to make a specific choice makes sure they know what they are choosing, at least in theory.